doc/mf6io/mf6ivar/tex/gwe-nam-desc.tex

```diff
@@ -0,0 +1,25 @@
+% DO NOT MODIFY THIS FILE DIRECTLY.  IT IS CREATED BY mf6ivar.py 
+
+\item \textbf{Block: OPTIONS}
+
+\begin{description}
+\item \texttt{list}---is name of the listing file to create for this GWE model.  If not specified, then the name of the list file will be the basename of the GWE model name file and the '.lst' extension.  For example, if the GWE name file is called ``my.model.nam'' then the list file will be called ``my.model.lst''.
```

Comment:
Think the quotes around .lst need to be changed to latex style, which is `` ''.

Reply:
Fixed with 40f895c